.github/workflows/ci.yaml

```diff
@@ -1,27 +1,43 @@
-name: OAuth2 Client Tests
+name: CI
 
 on:
-    push:
-#        branches:
-#            - master
     pull_request:
-    release:
-
-#env:
-#    PHPUNIT_FLAGS: '-v'
-#    SYMFONY_REQUIRE: '>=2.8'
+    push:
+        branches: [ master ]
 
 jobs:
-    composer-validate:
```

Comment:
The `composer-validate` job needs to stay and should only be executed with the `--strict` flag. I'm ok with this job living in `static-analysis.yaml`